### ISSUE TITLE
inverted logic found on isExact

### ISSUE THREAD

**Issue by E3V3A** [SecUpwN/Android-IMSI-Catcher-Detector#561]
In the DbViewerFragment.java, the [isExact](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/blob/55bd49ccd8797d0619fe0b323c4c333a0925a695/app/src/main/java/com/SecUpwN/AIMSICD/fragments/DbViewerFragment.java#L322) has reverse logic, since it is using OCID's [changeable](http://wiki.opencellid.org/wiki/API#.22changeable.22_attribute) attribute. 

@banjaxbanjo How can we reverse the logic of this line: 
```java
"isExact: " + String.valueOf(tableData.getInt(tableData.getColumnIndex(DBTableColumnIds.DBE_IMPORT_IS_GPS_EXACT))), // isGPSexact
```

**Comment by banjaxbanjo** [comment]
@E3V3A I don't get what you're saying eva is isExact = changable?

**Comment by E3V3A** [comment]
No, it is: `isExact = !changable`. I.e. the opposite Boolean.

**Comment by banjaxbanjo** [comment]
Ah so this value from ocid is boolean 0 or 1. I thought it was int with random values. Do you want this to display true or false.

True if exact
Fasle if not??

**Comment by E3V3A** [comment]
Aooouuh man! :D I just want it to show the opposite of what it is already showing... `0 -> 1 and 1 -> 0`

**Comment by banjaxbanjo** [comment]
I'm a bit dumb sometimes😃.

You'd need to move it out of where its inserted and then do.

Int exact = tableData.getInt(tableData.getColumnIndex(DBTableColumnIds.DBE_IMPORT_IS_GPS_EXACT));

if (exact == 0){exact =1;}

Then do


"isExact: "+String.valueOf (exact),

**Comment by E3V3A** [comment]
Well if I could understand the masked java for the SQL you use, we can probably just do it there, with something like `NOT isExact` or something. Can you clarify how the SQL look for this?

**Comment by E3V3A** [comment]
@banjaxbanjo  
I don't like that solution, as it masks what is really in DB. This should be changed in the import processing of the CSV file, into DB. Posted temporary fix in: f2368168fd0c47be30ca3ca12a03ace732e5ff98

**Comment by SecUpwN** [comment]
>Posted temporary fix in: f2368168fd0c47be30ca3ca12a03ace732e5ff98

@E3V3A, why do you close that Issue with a "temporary fix" that you do not even like? Please re-open this if anything else needs to be done until this Issue is *really* fixed this the way it should be. Thank you!
